getting-started/data-node-authentication.md

```diff
@@ -0,0 +1,70 @@
+# Data Node Authentication
+
+In a multi-node environment, it is necessary to ensure that the access
+node have the necessary roles and permissions on all the data nodes
+where distributed hypertables are placed.
+
+This is critical since when TimescaleDB is executing a query or
+command it will use the same role on the data nodes for the
+fragments. This means that the same role need to exist on the data
+nodes and authenticate to them.
+
+In a typical setup, you will deploy the distributed hypertable in a
+database and use the same database on all the data nodes. This means
+that we need to ensure that a user that is connected to the access
+node can access the same databases on the data nodes.
+
+## Setting up password authentication
+
+In order to set up password authentication for a user `new_user`, it
```

Comment:
I think this section would benefit from being split into subsections about what needs to be done on (1) data nodes, and (2) the access node

Reply:
Split. Describing the setup for the data node first.